analyzer/src/main/kotlin/curation/Sw360PackageCurationProvider.kt

```diff
@@ -98,10 +98,8 @@ class Sw360PackageCurationProvider(config: Sw360StorageConfiguration) : PackageC
     private val connectionFactory = createConnection(config)
     private val releaseClient = connectionFactory.releaseAdapter
 
-    override fun getCurationsFor(packages: Collection<Package>): Map<Identifier, List<PackageCuration>> =
-        packages.mapNotNull { pkg ->
-            getCurationsFor(pkg.id).takeUnless { it.isEmpty() }?.let { pkg.id to it }
-        }.toMap()
+    override fun getCurationsFor(packages: Collection<Package>): List<PackageCuration> =
+        packages.flatMap { pkg -> getCurationsFor(pkg.id) }.distinct()
```

Comment:
I believe `distinct()` should not be added here to more closely resemble the previous code, and because it would be dropped in the next commit anyway.